paneldb/server/templates/baitsets.html

```diff
@@ -32,7 +55,34 @@ <h4>Upload a new bait set</h4>
 <h4>Available baitsets:</h4>
 
 <br><br>
-Submitted document: {{document}}
+Submitted document: {{document}} {{upload_folder}} <br>
+
+<div>
+        <table class="table table-striped">
+          <thead>
+            <tr>
+              <th scope=""col">Chromosome</th>
+              <th scope=""col">chr_start</th>
+              <th scope=""col">chr_stop</th>
+              <th scope=""col">length</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for line in bah %}
```

Comment:
Rename "bah" to something more professional maybe? :)

Print the baitset table only if there is something to show (if bah has some value). Otherwise when you load the page you just see the table header without content.
To do so you can introduce the an if expression

{% if bah (or whatever you name it) %}
Print all the table stuff
{% endif %}

Reply:
Ok, seems to work